packages/lumx-react/src/components/slideshow/Slideshow.tsx

```diff
@@ -182,18 +182,29 @@ export const Slideshow: Comp<SlideshowProps, HTMLDivElement> = forwardRef((props
         onChange(currentIndex);
     }, [currentIndex, onChange]);
 
+    const slideshowSlidesId = 'slideshow-slides';
+
     /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
     return (
-        <div
+        <section
             ref={mergeRefs(ref, setElement)}
             {...forwardedProps}
             className={classNames(className, handleBasicClasses({ prefix: CLASSNAME, theme }), {
                 [`${CLASSNAME}--fill-height`]: fillHeight,
                 [`${CLASSNAME}--group-by-${groupBy}`]: Boolean(groupBy),
             })}
-            tabIndex={0}
+            aria-roledescription="carousel"
+            aria-live={isAutoPlaying ? 'off' : 'polite'}
         >
-            <div className={`${CLASSNAME}__slides`}>
+            <div
+                id={slideshowSlidesId}
+                className={`${CLASSNAME}__slides`}
+                onMouseEnter={() => setIsAutoPlaying(false)}
+                onMouseLeave={() => setIsAutoPlaying(Boolean(autoPlay))}
+                onBlur={() => setIsAutoPlaying(Boolean(autoPlay))}
+                onFocus={() => setIsAutoPlaying(false)}
```

Comment:
This is where it may get a bit complicated (and can be tackled on a later PR), but this behavior should be applied to:
* The slide itself, as you did here
* Any interactive element within the slide, which may be a bit tricky...
* The controls (except the future pause button, if we follow the WAI guidelines).

Maybe we could add some sort of 'focusWithin' behavior ? :thinking: